ui/src/containers/Resources/index.tsx

```diff
@@ -72,10 +74,23 @@ const Resources: React.FC = observer(() => {
     );
   };
 
-  return resources.resources.size === 0 ? (
-    <Spinner className="hub-spinner" />
-  ) : (
-    <React.Fragment>{checkResources(resources.filteredResources)}</React.Fragment>
-  );
+  const checkResourceStatus = (resources: IResourceStore) => {
+    return resources.err === '' ? (
+      resources.resources.size === 0 && resources.status !== 200 && resources.status !== 404 ? (
+        <Spinner className="hub-spinner" />
+      ) : (
+        <React.Fragment>{checkResources(resources.filteredResources)}</React.Fragment>
+      )
+    ) : (
+      <EmptyState variant={EmptyStateVariant.full} className="hub-resource-emptystate__margin">
+        <EmptyStateIcon icon={WarningTriangleIcon} />
+        <Title headingLevel="h5" size="md">
```

Comment:
Not a blocker, should we add a className and include the style in .css file

Reply:
those are the props of Title(Patternfly component) and not a custom css , so we can't add by a className